databricks/koalas/frame.py

```diff
@@ -4581,6 +4582,246 @@ def describe(self, percentiles: Optional[List[float]] = None) -> 'DataFrame':
                                   index_map=[('summary', None)])
         return DataFrame(internal).astype('float64')
 
+    def _cum(self, func, skipna: bool):
+        if len(self._internal.index_columns) > 1:
+            raise ValueError("Index must be set.")
+
+        index_columns = self._internal.index_columns
+        data_columns = self._internal.data_columns
+        window = Window.orderBy(
+            index_columns).rowsBetween(Window.unboundedPreceding, Window.currentRow)
+        sdf = self._sdf
+
+        for column_name in data_columns:
+            # It defines another column that holds true or false for nulls first.
+            is_null_column = "%s_isnull" % column_name
```

Comment:
Actually .. I think those expressions should go to `Series.cum*` .. I will do it in the next release.